### PR TITLE
Stats by periods and last week purchases 

### PR DESCRIPTION
**Pull request by aarrtthhuurr**

### CODE REVIEW

File: src/components/Home/Balance.js [hunk 1]
```diff
@@ -31,6 +31,14 @@ export default class Balance extends React.PureComponent {
 						{t('no_balance')}
 					</Text>
 				)}
+				<Text style={{ fontSize: 13, color: colors.secondary }}>
+					{thisWeek ? (
+						<Text style={{ fontWeight: 'bold', color: colors.less }}>{thisWeek}</Text>
+					) : (
+						_('loading_text_replacement')
+					)}{' '}
+					{t('this_week')}
```
Comment (NastuzziSamy) [inline src/components/Home/Balance.js]: "0,00€ dépensés" here a problem, should be "dépensé" even for < 1 and if i have refilled, i got +20€ so you need to change the expression

Comment (aarrtthhuurr) [reply src/components/Home/Balance.js]: Nice found

Comment (NastuzziSamy) [reply src/components/Home/Balance.js]: Will be fixed by another PR

**Comment by NastuzziSamy** [comment]
67764337_2438226752933231_9054206499269640192_n

Final result :)

**Pull request merged**